terraform/README.md

```diff
@@ -0,0 +1,16 @@
+## Decal VM Terraform configuration
+
+There are two additional files containing sensitive
+information to use with this configuration,
+credentials.auto.tfvars and students.auto.tfvars.
```

Comment:
Maybe add an explanation of what these two sets of variables are used for? I'm assuming they overwrite variables below like the DigitalOcean token, etc, but having an explanation here would be good for anyone who is not familiar with Terraform.